pyproject.toml

```diff
@@ -27,9 +27,6 @@ ignore = [
     "keylime/da/examples/redis.py",
     "keylime/da/examples/file.py",
     "keylime/da/record.py",
-    "test/oldtest.py",
-    "test/test_ima_dm.py",
-    "test/test_restful.py",
```

Comment:
Didn't we propose to ignore all the tests? IMHO those should not be type checked ...

Reply:
Now it's done and having the test cases type-checked as well caused me to fix a missing field in the Policies type, which does have some benefits already.